Formula/hof.rb

```diff
@@ -7,6 +7,13 @@ class Hof < Formula
   license "BSD-3-Clause"
   head "https://github.com/hofstadter-io/hof.git", branch: "_dev"
 
+  # Latest release tag contains `-beta`, which is not ideal
+  # adding a livecheck block to check the stable release
```

Comment:
@chenrui333 For what it's worth, I only add a preceding `livecheck` block comment for uncommon scenarios that benefit from (or require) an explanation (i.e., something that's not immediately clear that we don't want to forget). Using the standard regex for Git tags like `1.2.3`/`v1.2.3` (`/^v?(\d+(?:\.\d+)+)$/i`) to avoid unstable tags in a Git repository is a really common situation, so it doesn't warrant a comment in the formula from my perspective (it just adds noise). When I see a `livecheck` block checking Git tags and using this regex, it's implied that it was added to avoid tags that we don't want (as we try to only add a `livecheck` block when it's necessary).

In this scenario, I explain the situation in the PR description instead. This makes it available to anyone searching pull requests for information on the `livecheck` block (e.g., why it was added/modified in this way) without adding unnecessary noise to the formula file.

That said, thanks for this fix!